src/models/schoolDays.ts

```diff
@@ -0,0 +1,67 @@
+import mongoose from 'mongoose'
+const { Schema } = mongoose
+
+const schoolDays = new Schema({
+  studentId: {
+    type: Schema.Types.ObjectId,
+    ref: 'Students',
+    unique: true,
+  },
+  lrn: {
+    type: String,
+    unique: true,
+    minlength: 12,
+    maxlength: 12,
+  },
+  attendanceId: {
+    type: Schema.Types.ObjectId,
+    ref: 'Attendances',
+    unique: true,
+  },
+  academicYear: {
+    from: String,
+    to: String,
+    required: true,
+  },
+  gradeLevel: {
+    type: String,
+    enum: ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10'],
+    required: true,
+  },
+  daysOfSchool: {
```

Comment:
Redundant, ilagay mo yung mga months directly, wag mo ilagay sa loob ng days of school